### PR TITLE
Random header

### PR DESCRIPTION
**Pull request by toakes59**
added random header to top of page.  Easier to see then the title table

- [ ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/cplusplus/random-functions/index.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 title: Random Functions
 ---
 
+#Random Functions
```
Comment (thecodingaviator) [inline guide/english/cplusplus/random-functions/index.md]:
```suggestion
# Random Functions
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

File: guide/english/cplusplus/random-functions/index.md (hunk 1 above)
Comment (thecodingaviator) [reply guide/english/cplusplus/random-functions/index.md]: Make this change

**Comment by RandellDawson** [comment]
@thecodingaviator This PR has gone stale.

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**